### PR TITLE
OpenSHMEM 1.6rc2 Draft

### PR DESCRIPTION
**Pull request by jdinan**
This PR contains all proposed section committee changes for the OpenSHMEM 1.6rc1 draft, including the following:

* #547 
* #552 
* #548 
* #551
* #549 
* #543
* #553 

PDF: [OpenSHMEM-1.6rc2.pdf](https://github.com/user-attachments/files/17347028/OpenSHMEM-1.6rc2.pdf)

### CODE REVIEW

File: content/coverpage.tex [hunk 1]
```diff
@@ -60,6 +60,7 @@ \subsection*{\openshmem 1.6}
 \item Aurelien Bouteiller, \ac{UTK}
 \item Mark Brown, Intel
 \item Bob Cernohous, \ac{HPE}
+\item Matthew Davis, Georgia Tech
```
Comment (ahayashi) [inline content/coverpage.tex:63]: @ronawho, could you please add `Youssef Elmougy, Georgia Tech` (@youssefelmougy)?

Comment (ronawho) [reply content/coverpage.tex:63]: @jdinan can you add contributors as a doc edit or do I need to add them? If I need to add, is that something that goes in the frontmatter RC2 or a new PR?

File: content/coverpage.tex [hunk 2]
```diff
@@ -60,6 +60,7 @@ \subsection*{\openshmem 1.6}
...
 \item James Dinan\footnotemark[1], NVIDIA
```
Comment (jdinan) [inline content/coverpage.tex:64]:
```suggestion
\item James Dinan\footnotemark[1], NVIDIA
\item Youssef Elmougy, Georgia Tech
```

File: content/coverpage.tex (hunk 1 above)
Comment (jdinan) [reply content/coverpage.tex:63]: @ahayashi Thank you! I'll include this change as a doc. edit before we publish the final draft.

File: content/coverpage.tex (hunk 2 above)
Comment (ahayashi) [reply content/coverpage.tex:64]: Thank you!

**Pull request merged**